src/Controller/DefaultController.php

```diff
@@ -117,34 +110,31 @@ public function releaseJson(): Response
      * @Route("/release-notes/", methods={"GET"})
      * @Route("/release-notes/{version}", methods={"GET"}, name="release-notes-for-version")
      * @Route("/release-notes/{folder}/{version}", methods={"GET"}, name="legacy-release-notes-for-version")
-     *
-     * @param \Symfony\Component\HttpFoundation\Request $request
-     * @param string $version
-     * @return \Symfony\Component\HttpFoundation\Response
      */
     public function releaseNotes(Request $request, string $version = ''): Response
```

Comment:
@Crell would you also prefer `?string` here?

Reply:
Looks like this is the official convention see https://symfony.com/doc/current/routing.html#optional-parameters